epub33/a11y/index.html

```diff
@@ -669,8 +669,8 @@ <h5>Applicability</h5>
 							any additional requirements beyond those defined in [[EPUB-3]] to be conformant with this
 							specification.</p>
 
-						<p>To improve the usability of Media Overlays, however, EPUB Creators should meet the <a
-								href="#sec-mo-obj">objectives defined in this section</a>.</p>
```

Comment:
This paragraph can be rephrased as:
However to maximize the effectiveness of Media Overlays for people with different disabilities, EPUB Creators are recommended to meet the objectives defined in the next section.

Reply:
I know we're now adhering to uppercase RFC keywords being normative and lowercase not, but having should or recommended in the same sentence as normative explanation is inviting confusion. In this case, I think we're better off sticking with "strongly encouraged", so I've tweaked the sentence to:

"To maximize the effectiveness of Media Overlays for people with different disabilities, however, EPUB Creators are strongly encouraged to meet the OPTIONAL objectives defined in the next section."